table/server/common/src/main/java/alluxio/master/table/DatabaseInfo.java

```diff
@@ -47,7 +49,11 @@ public DatabaseInfo(String location, String ownerName, PrincipalType ownerType,
     mOwnerName = ownerName;
     mOwnerType = ownerType;
     mComment = comment;
-    mParameters = params;
+    if (params == null) {
+      mParameters = Collections.emptyMap();
+    } else {
+      mParameters = params;
+    }
   }
```

Comment:
The glue UDB handles the NPE for parameters, do we want to check it again here?

Reply:
The reason I put that here, was because this field was not classified as `@Nullable`, so I am enforcing that here.